apps/deploy-web/src/utils/apiUtils.ts

```diff
@@ -66,82 +66,62 @@ export class ApiUrlService {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/addresses/${address}/transactions/${skip}/${limit}`;
+    return `${this.baseApiUrl}/v1/addresses/${address}/transactions/${skip}/${limit}`;
   }
   static addressDeployments(address: string, skip: number, limit: number, reverseSorting: boolean, filters: { [key: string]: string }) {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/addresses/${address}/deployments/${skip}/${limit}${appendSearchParams({ reverseSorting, ...filters })}`;
+    return `${this.baseApiUrl}/v1/addresses/${address}/deployments/${skip}/${limit}${appendSearchParams({ reverseSorting, ...filters })}`;
   }
   static graphData(snapshot: string) {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/graph-data/${snapshot}`;
+    return `${this.baseApiUrl}/v1/graph-data/${snapshot}`;
   }
   static providerGraphData(snapshot: string) {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/provider-graph-data/${snapshot}`;
+    return `${this.baseApiUrl}/v1/provider-graph-data/${snapshot}`;
   }
   static blocks(limit: number) {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/blocks${appendSearchParams({ limit })}`;
+    return `${this.baseApiUrl}/v1/blocks${appendSearchParams({ limit })}`;
   }
   static providerActiveLeasesGraph(providerAddress: string) {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/provider-active-leases-graph-data/${providerAddress}`;
+    return `${this.baseApiUrl}/v1/provider-active-leases-graph-data/${providerAddress}`;
   }
   static providerAttributesSchema() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/provider-attributes-schema`;
+    return `${this.baseApiUrl}/v1/provider-attributes-schema`;
   }
   static networkCapacity() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/network-capacity`;
+    return `${this.baseApiUrl}/v1/network-capacity`;
   }
   static gpuModels() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/internal/gpu-models`;
+    return `${this.baseApiUrl}/internal/gpu-models`;
   }
-  // Github
   static auditors() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/auditors`;
+    return `${this.baseApiUrl}/v1/auditors`;
   }
-  static mainnetNodes() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/nodes/mainnet`;
-  }
-  static testnetNodes() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/nodes/testnet`;
-  }
-  static sandboxNodes() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/nodes/sandbox`;
-  }
-  static mainnetVersion() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/version/mainnet`;
-  }
-  static testnetVersion() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/version/testnet`;
-  }
-  static sandboxVersion() {
-    return `${browserEnvConfig.NEXT_PUBLIC_API_BASE_URL}/v1/version/sandbox`;
+
+  static get baseApiUrl() {
```

Comment:
fixes a bug introduced by me earlier - api url should vary based on the network for the above endpoints